src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,6 +124,37 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
+        
+        /**
+         * Represents the 'Export' button	

```

Comment:
Code comments are a maintenance burden, so we should have just the comments that are really essential. (More often than not, code comments get out of synch with the code they are commenting, leading to other maintenance issues.)
Code speaks for itself, in most of the cases ;)